forte/processors/data_augment/utils/utils.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
```

Comment:
Most functions here do not need to be re-implmented, please try to use the wheels, not reinventing them.